### PR TITLE
Fix a few misc. `ltijs` typing issues (#71)

### LINKED ISSUE #71: Fix `ltijs` typings
I discovered a few issues that the linting configuration in VSCode is flagging in `server/localTypes/ltijs`. The `Express` type needs to be declared, `UserInfo` needs to be defined, and an instance of `Record<>` needs arguments. I'll open a PR to fix them.

### PR DESCRIPTION
**Pull request by ssciolla**
It looks like I missed a few things that the linter is now complaining about in `server/localTypes/ltijs`. They should be addressed here. The PR will resolve #71.

### CODE REVIEW

File: ccm_web/server/localTypes/ltijs/index.d.ts [hunk 1]
```diff
@@ -122,6 +123,13 @@ declare module 'ltijs' {
...
   }
 
+  interface UserInfo {
+    given_name: string
+    family_name: string
+    name: string
+    email: string
+  }
+
   interface IdToken {
```
Comment (ssciolla) [inline ccm_web/server/localTypes/ltijs/index.d.ts:133]: @pushyamig, do you want me to add the `custom` key-value pair here as well?

Comment (pushyamig) [reply ccm_web/server/localTypes/ltijs/index.d.ts:133]: Yeah, Make sense to have it if you are touching this code. Here is how I did it? https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/66/files#diff-4be8caff9c88221f677afdb7f6238e2ffb00f1a23a40c1de9720c2a6a0a97e25

**Review by pushyamig — APPROVED** [review]

File: ccm_web/server/localTypes/ltijs/index.d.ts (hunk 1 above)
Comment (ssciolla) [reply ccm_web/server/localTypes/ltijs/index.d.ts:133]: K, I'll probably get to this tomorrow. Thanks

Comment (ssciolla) [reply ccm_web/server/localTypes/ltijs/index.d.ts:138]: @pushyamig, please take a look at this commit: https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/72/commits/1374c30e01b5ea0de55c2fa6ab8d728ce55364c3

I based these changes on the ltijs docs version of the "idToken" and "ContextToken" here: https://cvmcosta.me/ltijs/#/provider?id=authentication-and-routing I also compared them to what Canvas was sending. I guess I assumed that what was in the published types before (and other WIP) I saw was still up to date.

Comment (pushyamig) [reply ccm_web/server/localTypes/ltijs/index.d.ts:138]: I see that you try to get to bottom of it for better understanding. Nice to see you are thinking that way.

**Review by pushyamig — APPROVED** [review]
Looks Good!

**Comment by ssciolla** [comment]
@pushyamig, thanks for the review! I think there are still some issues with other parts of the types, but this should resolve some of the warnings I'm seeing. I'm going to merge this and will file separate PRs as necessary to fix other things.

**Comment by pushyamig** [comment]
@ssciolla, I do  see some error in the typing file for ltijs. Is this something can be resolved or we need to adjust our tsconfig.json to added this as exception?

Screen Shot 2022-01-25 at 12 41 03 PM

**Pull request merged**